Parser/src/Parser.cpp

```diff
@@ -8,518 +8,12 @@
 #include "../includes/Parser.h"
 
 Parser::Parser(char* filename) {
-	this->currentCodeSnippet = this->composeTable(filename);
+	this->scanner = new Scanner (filename);
 	ParseTree::initStatic();
 }
 
-ParseTree* Parser::parse() {
-	if (this->currentCodeSnippet == nullptr) {
```

Comment:
The Parser will be redone from scratch. Only (major) parts of the ParseTree (the postprocessing, i.e. `checkType()` and `makeCode()`) will remain.